examples/practice-typescript/src/components/TableHeader /index.tsx

```diff
@@ -0,0 +1,40 @@
+import User from 'scripts/types/user';
+import './index.scss';
+import Product from 'scripts/types/product';
+
+interface HeaderItemProps {
+  label: string;
+}
+
+const HeaderItem = ({ label }: HeaderItemProps) => {
+  const formatLabel = (str: string) => {
```

Comment:
For things that do not depend on props, states, or components, like this function, you should move outside the component.

Reply:
Fixed

Reply:
@thanhtrantruong47 not resolved yet

Reply:
Fixed